modules/distribution/handle_block.go

```diff
@@ -12,17 +12,18 @@ import (
 
 	"github.com/forbole/bdjuno/database"
 	distrutils "github.com/forbole/bdjuno/modules/distribution/utils"
+	"github.com/forbole/bdjuno/types/config"
 )
 
 // HandleBlock represents a method that is called each time a new block is created
-func HandleBlock(block *tmctypes.ResultBlock, client distrtypes.QueryClient, db *database.Db) error {
+func HandleBlock(block *tmctypes.ResultBlock, client distrtypes.QueryClient, db *database.Db, cfg *config.Config) error {
```

Comment:
Can we have the `cfg` parameter as the first parameter in order to keep it consistent with other methods as well? I tend to always use the same order of parameters: 

1. configuration objects
2. block/tx/message
3. query clients
4. codecs
5. database

The same also applied elsewhere

Reply:
Ok 👍🏻